src/main/java/emu/grasscutter/game/talk/TalkManager.java

```diff
@@ -27,5 +29,15 @@ public void triggerTalkAction(int talkId) {
         getPlayer().getQuestManager().queueEvent(QUEST_CONTENT_COMPLETE_ANY_TALK, talkId);
         getPlayer().getQuestManager().queueEvent(QUEST_CONTENT_COMPLETE_TALK, talkId);
         getPlayer().getQuestManager().queueEvent(QUEST_COND_COMPLETE_TALK, talkId);
+        saveTalkToQuest(talkId, talkData.getQuestId());
+    }
+
+    public void saveTalkToQuest(int talkId, int mainQuestId) {
+        // TODO, problem with this is that some talks for activity also have 
+        // quest id, which isnt present in QuestExcels
```

Comment:
For cases like this it would be good to also document an example, especially for files with this many entries.
Also maybe we could just solfe this by saving it to another map/array/table?